### DISCUSSION TITLE
"Suggestion for the Addition of an Ambidextrous Rank in osu!std"

### DISCUSSION THREAD

**Discussion by Aloux01757** [ppy/osu#22415] [Ideas] [answered]
I would like to suggest an addition to the osu!std ranking system, a exclusive rank for ambidextrous players. This would be an important accessibility option, as although most players are either right or left-handed, there is a small percentage of ambidextrous players in the world. Currently, these players are disadvantaged by having to choose between using their dominant hand or not, and this can negatively affect their scores in the ranking.

I believe that osu! could have three types of ranking: global ranking, right-handed and left-handed. The global ranking would be based on the player's dominant hand, while the right and left-handed rankings would be for players who prefer to use a specific hand. This would not be a way of "gaining an advantage," as the player would be training with the other hand, and this could help them improve their skill with both hands. Additionally, this would allow players who are not truly ambidextrous, but want to experiment with playing with both hands, without compromising the current ranking.

**Comment by mk56-spn** [comment]
This has to be the most ridiculously niche thing among niche things to ever niche. Not a Dev but I can pretty much say with certainty this won't be happening. 


The amount of effort this would require so vastly outweighs the hypothetical minimal benefits ( and I'd argue it comes with a plethora of negatives )

**Reply by Aloux01757** [reply]
"Specific niche? I understand that it may seem self-serving, but I believe it is more of an accessibility issue. I understand that it's a specific request, but instead of dismissing it as "ridiculous", I would like to hear your thoughts on how we can elaborate on this idea and make it viable. It seems a bit harsh."

**Reply by Rekunan** [reply]
There's not really any way to make the idea viable because the base goal is to make the rankings fair between players that have different physically medical situations, in this case handedness. This lends itself to a slippery slope of considering other situations, such as people who only have one fully working eye, as there's no real reason to consider this one above the others, and would therefore start an immense amount of effort and work that's regularly ongoing just to account for all the situations.

> Additionally, this would allow players who are not truly ambidextrous, but want to experiment with playing with both hands, without compromising the current ranking.

On this point, I don't really see how them experimenting would compromise that. [Matrix](https://osu.ppy.sh/users/5052899) comes to mind as a player that's learning to play with his other hands (due to wanting to learn with no bad muscle memory or something) and he seems to be doing fine: https://www.youtube.com/watch?v=eV8EcmUPEXI only stagnating in the rankings as he's moved on to focusing on other things, such as music creation.

> Currently, these players are disadvantaged by having to choose between using their dominant hand or not, and this can negatively affect their scores in the ranking.

As a non-ambidextrous person, I don't see how this is a disadvantage, they just have to choose and stick to one no?

**Reply by mk56-spn** [reply]
To add to what rekunan mentioned here if you look you'll see left handed players are over represented compared to population levels when it comes to top players. I'd wager this is due to lefties having to often use their right hand for many tools which simply aren't around commonly in left handed form . Which would explain them having more fine control with both hands than the average. With this in mind it is likely most of the difference in perceived difficulty would come from habit more than nature / genes, making this request null and void

**Reply by Aloux01757** [reply]
First of all, could you please show some respect? The language used is quite heavy and I, as an autistic person, don't understand non-literal language very well. For example, instead of saying "There's really no way to make the idea viable," you could say "I don't think this idea is viable." I apologize if my suggestion seemed like a snowball, it's not my intention to force anything. The same goes for "null and void."

Now, regarding the basic objective of making the ranking fair for players with different medical physical conditions, in this case handedness, and how it's similar to people who have only one fully functional eye, because there's no real reason to consider this above others. I find this information strange, as someone with vision problems doesn't have much to do with hands. I'm talking about skill, and are eyes  a skill? so how is it above others that I've mentioned?
"would therefore start an immense amount of effort and work that's regularly ongoing just to account for all the situations." Could you provide more examples of how other situations can be invalidated? I am concerned about the accessibility of this small minority, but without conflicting or unfairly using the ranking system, you know?

Additionally, this would allow players who are not truly ambidextrous, but want to experiment with playing with both hands, without compromising the current ranking.
On this point, I don't really see how them experimenting would compromise that. I mentioned that those who want to try playing with both hands for example, for fun, it would be ideal, so as not to compromise the ranking, which is the idea behind creating two rankings.

Currently, these players are disadvantaged by having to choose between using their dominant hand or not, and this can negatively affect their scores in the ranking.
As a non-ambidextrous person, I don't see how this is a disadvantage, they just have to choose and stick to one no?

An ambidextrous person has dominance in two hands, but depending on the activity and stimulation, one hand may perform better than the other, without compromising both sides of the body, for example, an ambidextrous person should have control over both feet on a ball, in the hands, and eyes (reflexes), it depends on the stimulation and how it's inserted, in this case, I'm talking about people who want to play with both hands in different situations, in short.

**Reply by Aloux01757** [reply]
"Both of you need to stop speaking in such a manner by constantly bringing up phrases like "slippery slope," "the most ridiculous niche," and "wager." If you have any doubts or questions, feel free to ask me and I will respond, but please refrain from using such vulgar language. Consider how others may perceive it if they were to read it on the internet."

**Comment by bdach** [comment]
I'm not sure how this would even be enforced. How would handedness be decided? Pinky promise? What's to stop people from, you know, lying about what hand they're playing with?

**Reply by Aloux01757** [reply]
Using satire is not only inappropriate, it can also be considered offensive and hurtful. Instead, please express your thoughts and ideas in a clear and respectful manner.

**Reply by Aloux01757** [reply]
...

**Reply by bdach** [reply]
I may have been a little facetious, but the question I'm asking is very real. Country leaderboards had similar problems with users gaming flag change requests, to the point that flag changes were stopped by the team.

This proposal is even worse in that respect, because I don't know how the integrity of those hand-specific leaderboards would be upheld without submitting medical information I guess? Which I imagine we *really* do not want to do.

As a final note, I notice that you have been pointing out people's language a lot, but not really addressing any of their actual points.

**Reply by Aloux01757** [reply]
Can you stop satire again? It gets a bit boring, so the first question because of the medical fact may be true that it is impossible to do this review, instead of saying that we don't want it, which seems like a lack of will, tell us what do you think we could address? like, it will be some algorithm for example: video test and another's form's, put some idea, about the very formal language I have and because I'm nervous "Autistic things", but I really don't want to cause harm to anyone or point it out. Excuse me.

**Reply by Aloux01757** [reply]
(I notice that you have been pointing out people's language a lot, but not really addressing any of their actual points.) I would like to communicate accurate and reliable information, especially for individuals with autism (me), this is not "done with evil".

**Reply by Theighlin** [reply]
> Can you stop satire again? It gets a bit boring

Nothing of what he said was satire in the second reply

> instead of saying that we don't want it, which seems like a lack of will, tell us what do you think we could address?

Nobody in this thread said they don't want it, it's just a very niche thing that very few people would be interested in. What he said he doesn't want to happen is for the osu team to ask users for and review their medical information.

> like, it will be some algorithm for example: video test and another's form's

In my humble opinon, such an algorithm wouldn't work. You could maybe train an AI on the gameplay of different handed streamers or youtubers and their replays to then tell right handed and left handed players apart, but even if that were possible, i don't think there's substantial enough difference between the 2 for it to work, and there is no one here with the time or even the capabilities to do that.

As has been stated before, it's impossible to practically make this work without huge amount of efforts, and asking the players for what hand they use wouldn't work, because they could be lying, maybe, to get in a higher spot on the lesser participated leaderboard of the 2. Even if you asked video proof of their playing style, they could fake that too.

And for last, personally, i don't think such leaderboards are needed.

**Reply by Aloux01757** [reply]
>Nothing of what he said was satire in the second reply

Yes, you are right. Now I realize that as an autistic person, I struggle with figures of speech, which caused confusion for me. I am trying to cope with my condition.

>Nobody in this thread said they don't want it, it's just a very niche thing that very few people would be interested in. What he said he doesn't want to happen is for the osu team to ask users for and review their medical information.

Is this the polite answer I wanted, although, I already understood that it is an extremely small niche, and not by rejecting it, I already knew, how to write calmly improves the way I express myself to you, I understand your concern about recognizing an ambidextrous person and in addition to medical information, but do you have alternatives? Because, like, how can someone like me take advantage of using two different hands? Do you have any suggestions? I mean, without applying what I'm talking about?

>like, it will be some algorithm for example: video test and another's form's

Not really reading word for word what I said, but do you have other ways you think, or ask for more details with me, you know? to have a corporate idea.

**Comment by Aloux01757** [comment]
I believe it is unjust when I speak politely in my comment and then someone like "mk56-spn" responds rudely and dismissively. I question how some developers can be so rude, especially when compared to the last person who responded to me. This behavior could negatively impact your reputation on the internet. Furthermore, I am an autistic person and I tend to take things literally and use formal language, but it is not out of malice. I am disappointed by the lack of empathy.

**Comment by peppy** [comment, marked as answer]
This is not going to happen. I think if there's enough interest, it could be implemented on an external site similar to other niche rankings.

**Reply by Aloux01757** [reply]
It's not the idea of concern that I have, whether it occurs or not. I am concerned with rudeness as it was responded...

**Reply by Aloux01757** [reply]
I believe that almost nobody is communicating in a healthy way (almost), this has already been stated in the first rude response. Again, my concern is not whether or not it occurred, but rather how it was handled. I did not like it!

**Reply by peppy** [reply]
Sorry you felt that way. I'm going to lock this conversation as we've already reached a unanimous opinion.
